plugins/postcss-global-data/src/index.ts

```diff
@@ -24,13 +24,13 @@ const creator: PluginCreator<pluginOptions> = (opts?: pluginOptions) => {
 			let importedCSS = new Set<ChildNode>();
 
 			return {
-				Once: (root) => {
+				Once: (root, { postcss }) => {
```

Comment:
I missed this before, but we can "extract" PostCSS from here.
This allows us to avoid importing PostCSS directly.